easybuild/easyconfigs/d/deepTools/deepTools-3.1.2-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,61 @@
+
+easyblock = "PythonPackage"
+
+name = 'deepTools'
+version = '3.1.2'
+
+homepage = 'https://pypi.python.org/pypi/deepTools/'
+description = """Tools to process and analyze deep sequencing data."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['fc8b17f0a791b5ba6a3d6bcb32ec0731614da2397d38218217ec3d3cd6ee6129']
+
+python = 'Python'
+pythonversion = '2.7.14'
+pyshortver = ".".join(pythonversion.split(".")[:-1])
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [
+    (python, pythonversion),
+    ('matplotlib', '2.1.2', versionsuffix),
+    ('Pysam', '0.14.1', versionsuffix),
+    ('cURL', '7.58.0'),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_list = [
+    ('plotly', '2.0.0', {
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/p/plotly/'],
+        'checksums': ['eede6581abe00f319909fa84e99f6033b9e735427fb64b570dfbc312fc0dd771'],
+    }),
+    ('py2bit', '0.3.0', {
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/p/py2bit/'],
+        'checksums': ['450555c40cba66957ac8c9a4b6afb625fb34c4bb41638de78c87661ff8b682ef'],
+    }),
+    ('pyBigWig', '0.3.4', {
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyBigWig/'], 
+        'checksums': ['8c97a19218023190041c0e426f1544f7a4944a7bb4568faca1d85f1975af9ee2'],
+        'modulename': 'pyBigWig',
+    }),
+]
+
+# specify that Bundle easyblock should run a full sanity check, rather than just trying to load the module
+full_sanity_check = True
+
+sanity_check_paths = {
+    'files': ['bin/bamCompare', 'bin/bamCoverage', 'bin/bamPEFragmentSize', 'bin/computeGCBias', 'bin/computeMatrix',
+              'bin/correctGCBias', 'bin/multiBamSummary', 'bin/plotCorrelation', 'bin/plotCoverage',
+              'bin/plotHeatmap', 'bin/plotProfile'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = ["python -c 'import deeptools'"]
```

Comment:
@ebirn This is done automatically (`import` test with lowercase software name), so you can remove this